src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -6,6 +6,8 @@
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <ProjectGuid>{43311AFB-D7C4-4E5A-B1DE-855407F90D1B}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Configuration Condition=" '$(Configuration)' == '' ">$(TargetOS)_Debug</Configuration>
```

Comment:
Setting this here isn't going to work as we always set these properties in dir.props.